.github/release.yml

```diff
@@ -10,3 +10,5 @@ changelog:
       labels: ['documentation']
     - title: 'Dependency Upgrades'
       labels: ['dependencies']
+    - title: 'Internal Changes'
+      labels: ['internal', 'github_action']
```

Comment:
Also, I believe the `internal` label does not provide much of additional information. Maybe we could use labels such as, `build`, `test`, `refactor` and we know those are all internal changes. Wdyt?

Reply:
Hmmm this is actually a very good point! I like it a lot I always like things to be classified in a clear and organized way! Will look into this and create labels if needed. We'll need to run this through the team so they are aware of the new labels.